sdk/src/main/scala/com/horizen/AbstractSidechainNodeViewHolder.scala

```diff
@@ -132,6 +135,17 @@ abstract class AbstractSidechainNodeViewHolder[
       }
   }
 
+  protected def processGenerateSecret: Receive = {
+    case AbstractSidechainNodeViewHolder.ReceivableMessages.GenerateSecret(secretCreator) =>
```

Comment:
Apparently no one is using this case class. Is there any code not yet pushed on this branch?

Reply:
Done